Userland/Applets/Audio/main.cpp

```diff
@@ -87,14 +87,32 @@ class AudioWidget final : public GUI::Widget {
         m_slider->set_max(100);
         m_slider->set_page_step(5);
         m_slider->set_step(5);
-        m_slider->set_value(m_slider->max() - m_audio_volume);
         m_slider->set_knob_size_mode(GUI::Slider::KnobSizeMode::Proportional);
         m_slider->on_change = [&](int value) {
             m_audio_volume = m_slider->max() - value;
-            double volume = clamp(static_cast<double>(m_audio_volume) / m_slider->max(), 0.0, 1.0);
+            double volume = clamp(static_cast<double>(m_audio_volume) / 100, 0.0, 1.5);
             m_audio_client->set_main_mix_volume(volume);
             update();
         };
+        if (m_audio_volume > 100) {
+            m_audio_volume_raised = true;
+            m_slider->set_max(150);
+        }
+        m_slider->set_value(m_slider->max() - m_audio_volume);
+
+        m_volume_raised_box = m_root_container->add<GUI::CheckBox>("\u2B06");
+        m_volume_raised_box->set_fixed_size(27, 16);
+        m_volume_raised_box->set_checked(m_audio_volume_raised);
+        m_volume_raised_box->set_tooltip(m_audio_volume_raised ? "Volume raised" : "Volume normal");
+        m_volume_raised_box->on_checked = [&](bool is_audio_volume_raised) {
+            if (is_audio_volume_raised)
+                m_slider->set_max(150);
+            else
+                m_slider->set_max(100);
+            m_audio_client->set_main_mix_volume(1.0);
```

Comment:
This should probably set the possibly newly-clamped volume, and not a constant 1.
```suggestion
            m_audio_volume = m_slider->max() - m_slider->value();
            m_audio_client->set_main_mix_volume(static_cast<double>(m_audio_volume) / 100.);
```